### PR TITLE
Added power support for the travis.yml file with ppc64le.

### PR DESCRIPTION
**Pull request by sreekanth370**
Added power support for the travis.yml file with ppc64le. This is part of the Ubuntu distribution for ppc64le. 
This helps us simplify testing later when distributions are re-building and re-releasing.

### CODE REVIEW

**Review by gururajrkatti — CHANGES_REQUESTED** [review]
What is the reason we are excluding jobs from amd ?

**Comment by gururajrkatti** [comment]
@sreekanth370 any updates on this ?

**Comment by sreekanth370** [comment]
Hi Guru ,   may be excluded the jobs for unsupported versions like python 2.6,3.2,3,3and pypy on both amd and ppc .
 kindly look which are failed version .
 https://travis-ci.com/github/sreekanth370/retrying/builds/212651280

**Comment by gururajrkatti** [comment]
You can exclude for only ppc if it is failing only for ppc

**Comment by sreekanth370** [comment]
excluded the  ppc  jobs , which are not support.
amd jobs fails for python 2.6, 3.2 and 3.3 .

**Comment by sreekanth370** [comment]
https://travis-ci.com/github/sreekanth370/retrying/builds/216787634

**Comment by gururajrkatti** [comment]
If you are excluding for both amd and ppc, then you can directly remove those versions. Add a comment with the reason for removing

**Comment by sreekanth370** [comment]
updated the PR , kindly check it

File: travis-ymls/retrying.py_travis.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  - 2.7
+  - 3.2
+  - 3.3
+  - 3.4
+  - 3.5
+  - pypy
+ # Disable unsupport version 2.6,3.2, 3.3 & pypy on ppc64le
+jobs: 
+  exclude:
+    - arch: amd64
```
Comment (gururajrkatti) [inline travis-ymls/retrying.py_travis.yml:27]: Can we do not do something like below and avoid excluding 2.6, 3.2 and 3.3 for both ?
python:
    - 2.7
    - 3.4
    - 3.5
    - pypy

**Review by gururajrkatti — CHANGES_REQUESTED** [review]

**Comment by sreekanth370** [comment]
unfortunately travis file was deleted .hence closing.

**Pull request closed without merging**